target/linux/ath79/image/generic.mk

```diff
@@ -965,6 +966,19 @@ define Device/yuncore_a770
 endef
 TARGET_DEVICES += yuncore_a770
 
+define Device/yuncore_cpe830
+  ATH_SOC := qca9531
+  DEVICE_VENDOR := YunCore
+  DEVICE_MODEL := CPE830(D)
+  DEVICE_ALT0_VENDOR = KuWfi
+  DEVICE_ALT0_MODEL = CPE830(D)
+  IMAGE_SIZE := 16000k
+  IMAGES += tftp.bin
+  IMAGE/tftp.bin := $$(IMAGE/sysupgrade.bin) | yuncore-tftp-header-qca953x
```

Comment:
This PR is blocked until my PR is through

Reply:
So change your PR so it fits the needs not only your wishes